apsi-enrollment-be/Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM gradle:6.3-jdk11
+WORKDIR /usr/src/app
+COPY . .
+RUN gradle wrapper
+RUN ./gradlew build
+
+FROM openjdk:11-jre-slim
```

Comment:
Czy potrzebujemy tutaj multi stage buildu? Gradle nie jest od razu z javką (chyba, że jest to jakaś stara java, to w takim wypadku można zamknąć)? Bo tak to można by się zamknąć do prostego `gradlew bootRun` lub też innej komendy, którą zapewnia nam spring.

Reply:
Można uruchamiać przez graldew, ale przy takim scenariuszu mamy mniejszy rozmiar obrazu na produkcji.